impl/src/main/java/com/sun/faces/context/UrlBuilder.java

```diff
@@ -16,21 +16,18 @@
 
 package com.sun.faces.context;
 
-import java.io.UnsupportedEncodingException;
 import java.net.URLEncoder;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Iterator;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
+import java.nio.charset.Charset;
+import java.util.*;
```

Comment:
Can you please reconfigure your IDE to not do that anymore? Wildcard imports are highly discouraged.